### PR TITLE
Mada ole md wiki

### PR DESCRIPTION
**Pull request by MadagascarOLE**
[Rawgit link](https://rawgit.com/MadagascarOLE/MadagascarOLE.github.io/Mada-ole-md-wiki/#!pages/profiles/MadagascarOLE.md)

### CODE REVIEW

**Comment by dogi** [comment]
It would be more awesome if you link to something :)

_Sent from my Google Pixel using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by MadagascarOLE** [comment]
i added the requested link :)

File: pages/profiles/MadagascarOLE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Hello All !!!!
+---
```
Comment (Yurockkk) [inline pages/profiles/MadagascarOLE.md]: By using Heading, you will get a free horizontal rule (check out [here](http://open-learning-exchange.github.io/#!pages/githubandmarkdown.md#Prerequisites)) and that's the reason why there are two horizontal rules on your [rawgit](https://rawgit.com/MadagascarOLE/MadagascarOLE.github.io/Mada-ole-md-wiki/#!pages/profiles/MadagascarOLE.md). Please delete line 2. Other than that, it looks good  to me 👍

Comment (MadagascarOLE) [inline pages/profiles/MadagascarOLE.md]: OK, i deleted the line 2 already

**Pull request merged**